javascript/javascript3/week3/promises.js

```diff
@@ -0,0 +1,52 @@
+//soujanyapolavarapu  sandhana14   cph-kiwi
+/* 
+Fetch all the 3 classmates repositories at the same time using Promise.all. Remember the all at once
+ exercise here?
+
+When you have the data for the different repositories, render the fullname of the repo, url of the repo, 
+and the owner of the repo.
+ See github-repos as an example of how the renderered repos should look. You are more than welcome to style 
+ it a bit nicer! */
+const url1 = "https://api.github.com/search/repositories?q=user:soujanyapolavarapu";
+const url2 = "https://api.github.com/search/repositories?q=user:qamarfj";
+const url3 = "https://api.github.com/search/repositories?q=user:cph-kiwi";
+
+
+class GetClass {
+    constructor(url) {
+        this.url = url;
+
+    }
+    get() {
+        fetch(this.url)
+            .then(response => response.json())
+            .then((studentData) => {
```

Comment:
The method called `get()` is doing both fetching and rendering at the same time. Consider splitting the part that gets the data (fetching) and the rendering (DOM manipulation)

Reply:
done by adding new function 
static randerData(studentData)